src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
Definitely need to add those typings. Wanna pair on getting the types for this function in order?